container/docker/factory.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/google/cadvisor/fs"
 	info "github.com/google/cadvisor/info/v1"
 
-	"github.com/fsouza/go-dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
```

Comment:
this should not be necessary. I suspect that your goimports binary is causing trouble.